src/common/docker-compose/index.ts

```diff
@@ -389,8 +389,8 @@ export class DockerComposeUtils {
       const line_parts = line.match(/(?:[^\s"]+|"[^"]*")+/g) || [];
       let name = line_parts[0];
       // Remove env name and counter: cloud_gateway_1
-      name = name.substring(name.indexOf('_') + 1);
-      name = name.substring(0, name.lastIndexOf('_'));
+      name = name.substring(name.indexOf('-') + 1);
```

Comment:
I didn't make any name changes yet. The logs command works for me locally on master without these changes.

Reply:
Alright will pull back this fix for now but should deploy the account fix.